examples/Python/ChatApp/README.md

```diff
@@ -0,0 +1,84 @@
+# Azure App Configuration - Python ChatApp Sample
+
+This sample demonstrates using Azure App Configuration to configure Azure OpenAI settings for a chat application built with Python.
+
+## Features
+
+- Integrates with Azure OpenAI for chat completions
+- Dynamically refreshes configuration from Azure App Configuration
+
+## Prerequisites
+
+- Python 3.8 or later
+- An Azure subscription with access to:
+  - Azure App Configuration service
+  - Azure OpenAI service
+- Required environment variables:
+  - `AZURE_APPCONFIG_ENDPOINT`: URL of your Azure App Configuration instance
+  - `AZURE_OPENAI_API_KEY`: API key for Azure OpenAI (optional if stored in Azure App Configuration)
+
+## Setup
+
+1. Clone the repository
+2. Install the required packages:
+   ```bash
+   pip install -r requirements.txt
+   ```
+3. Configure your Azure App Configuration store with these settings:
+   ```
+   ChatApp:AzureOpenAI:Endpoint - Your Azure OpenAI endpoint URL
+   ChatApp:AzureOpenAI:DeploymentName - Your Azure OpenAI deployment name
+   ChatApp:AzureOpenAI:ApiVersion - API version for Azure OpenAI (e.g., "2023-05-15")
+   ChatApp:AzureOpenAI:ApiKey - Your Azure OpenAI API key (preferably as a Key Vault reference)
```

Comment:
This should be optional. See https://github.com/Azure/AppConfiguration/blob/main/examples/DotNetCore/ChatApp/README.md